rust/rfprop/src/sigserve.cc

```diff
@@ -51,4 +56,61 @@ Report handle_args(int argc, char *argv[])
     return report;
 }
 
+PointToPointReport point_to_point(double tx_lat, double tx_lon, double tx_antenna_alt_m, double rx_lat, double rx_lon,
+                                  double rx_antenna_alt_m, double freq_hz, bool normalize, bool metric)
+{
+    double _min_lat = std::floor(std::min(tx_lat, rx_lat));
+    double _max_lat = std::floor(std::max(tx_lat, rx_lat));
+    double _tx_lon = std::floor(-tx_lon);
+    double _rx_lon = std::floor(-rx_lon);
```

Comment:
These negations probably only work because I'm testing in the western hemisphere. Figure out how to do it properly,

Reply:
Maybe describing the intent, or adding a test, would help?

Reply:
I think this has become too simplified compared to the original